lib/stink_bomb/configuration.rb

```diff
@@ -6,8 +6,31 @@ def initialize(options = {})
       self.message = options.fetch(:message) { default_message }
     end
 
+    def raise=(error_class)
+      case error_class
+      when true
+        add_bomb(RaiseBomb.new)
+      when false
```

Comment:
K.  I guess my preference would be for the `remove_bomb` at the top then (like in your comment).